static/scss/partials/main.scss

```diff
@@ -428,9 +388,9 @@ button:active {
 
 header {
   color: $color-white;
-  border-bottom: 1px solid var(--inkDark);
+  border-bottom: 1px solid rgba($color-grey-60, .5);
   background: var(--darkBlueGradient);
-  box-shadow: 0 0 3px 0 var(--inkDark);
+  box-shadow: 0 0 3px 0 rgba($color-grey-60, .5);
```

Comment:
nit: Not sure if we want to add a CSS linting rule re: prefixed zeros. I see we have `.5` here, but `0.5s` below on L404.